.github/workflows/integration.yml

```diff
@@ -4,8 +4,7 @@ on:
     tags:
       - v*
     branches:
-      - main
-      - release-*
+      - multi-storage
```

Comment:
I think this has to be reverted, right?

Reply:
yes, there is a commit that I need to remove. It is to have the CI running on this branch